questdb-rs/README.md

```diff
@@ -1,15 +1,17 @@
 # QuestDB Client Library for Rust
 
-Official Rust client for [QuestDB](https://questdb.io/), an open-source SQL database designed to process time-series data, faster.
+Official Rust client for [QuestDB](https://questdb.io/), an open-source SQL
+database designed to process time-series data, faster.
 
-The client library is designed for fast ingestion of data into QuestDB via the InfluxDB Line Protocol (ILP).
+The client library is designed for fast ingestion of data into QuestDB via the
+InfluxDB Line Protocol (ILP).
 
 * [QuestDB Database docs](https://questdb.io/docs/)
 * [ILP docs](https://questdb.io/docs/reference/api/ilp/overview/)
```

Comment:
nit: consider changing 'ILP docs' to something like: "InfluxDB Line Protocol Documentation". 
In general, I would the ILP acronym when possible.

Reply:
<img width="823" alt="image" src="https://github.com/questdb/c-questdb-client/assets/158619/c789c377-dfe1-47db-b75b-8c03ed795868">

Reply:
agree - I would keep ILP to a minimum and replace with InfluxDB Line Protocol, people do not know what ILP means